scripts/gitlab.rb

```diff
@@ -0,0 +1,4 @@
+registry_external_url 'http://127.0.0.1:5050'
+registry['enable']                    = true
+registry_nginx['ssl_certificate']     = "/etc/gitlab/ssl/example.pem"

```

Comment:
The files need to be renamed here, too.

On that note: should we have a test for the correct setup of the instance? Instead of having test failures later on? We can also do this outside of this PR to unblock things ...

Reply:
I'm thinking that might not be a bad idea.  Would we do it in the `await-healthy.sh` or the `healthcheck-and-setup.sh`

Reply:
Maybe there or in an acceptance test. Let's do this out of scope of this PR to unblock the other things. I'll open an issue.